apps/style/common.scss

```diff
@@ -389,6 +397,9 @@ html[dir='rtl'] #codeWorkspace {
   #visualizationColumn.responsive {
     max-width: $width;
   }
+  .small-footer.responsive {
```

Comment:
Something to keep an eye out for maybe we can do the next feature touching this, feels like there's a nice opportunity to consolidate some of this with sass. Potentially with an e.g. `@mixin size-visualization-area($width)`